src/args.rs

```diff
@@ -54,15 +56,21 @@ pub fn parse_args() -> Result<Option<Config>, Error> {
         })
         .ok_or("unable to determine home directory")?;
     let clipboard = pargs.contains(["-c", "--clipboard"]);
+    let copy_path = pargs.contains(["-p", "--copy-path"]) || env::var_os("GRAB_COPY_PATH").is_some();
 
     if clipboard && !SUPPORTS_CLIPBOARD {
         return Err("this git-grab was not built with clipboard support.")?;
     }
 
+    if copy_path && !SUPPORTS_CLIPBOARD {
+        return Err("this git-grab was not built with clipboard support.")?;
+    }
```

Comment:
This can be combined with the previous test: `if (clipboard || copy_path) && !SUPPORTS_CLIPBOARD`